pkg/cluster/internal/create/actions/cluster/templates/aws.eks.tmpl

```diff
@@ -70,6 +70,13 @@ spec:
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: MachineDeployment
 metadata:
+  annotations:
+  {{- if ne $node.NodeGroupMaxSize "" }} 
+    cluster.x-k8s.io/cluster-api-autoscaler-node-group-max-size: '{{ $node.NodeGroupMaxSize }}'  
+  {{- end }}
+  {{- if ne $node.NodeGroupMinSize "" }} 
+    cluster.x-k8s.io/cluster-api-autoscaler-node-group-min-size: '{{ $node.NodeGroupMinSize }}' 
+  {{- end }}
```

Comment:
max and min values should be divided by 3 (number of zones), perhaps with a new function..

Reply:
OK